spec/defines/init_spec.rb

```diff
@@ -99,20 +101,20 @@ def check_exec_sync(name, attributes)
               major: 7
             }
           },
-          systemd: true
+          systemd: true,
+          operatingsystem: 'RedHat',
+          osfamily: 'RedHat'
         }
       end
 
       check_file_set_header(
```

Comment:
this looks a bit funky.  Is it why `EmptyExampleGroup` fires?  Should you be using shared examples and `it_behaves_like` instead??

Reply:
this whole testsetup is a bit funky. Should we keep it as-is for now and refactor it later? Otherwise the diff gets even bigger.